docs/update_repo.md

```diff
@@ -0,0 +1,37 @@
+# How to Update Repository
+
+This repository is a branded version of the [foreman-operations-collection](https://github.com/theforeman/foreman-operations-collection) and requires some consideration when updating.
```

Comment:
no it isn't ;-)
(I am not going to comment this on every line where it talks about *operations*, but you get the idea ;-))

Reply:
:facepalm: